src/Service/CognitoIdentityProvider/src/CognitoIdentityProviderClient.php

```diff
@@ -318,6 +320,55 @@ public function adminInitiateAuth($input): AdminInitiateAuthResponse
         return new AdminInitiateAuthResponse($response);
     }
 
+    /**
+     * Resets the specified user's password in a user pool as an administrator. Works on any user.
+     *
+     * @see https://docs.aws.amazon.com/cognito-user-identity-pools/latest/APIReference/API_AdminResetUserPassword.html
+     * @see https://docs.aws.amazon.com/aws-sdk-php/v3/api/api-cognito-idp-2016-04-18.html#adminresetuserpassword
+     *
+     * @param array{
+     *   UserPoolId: string,
+     *   Username: string,
+     *   ClientMetadata?: array<string, string>,
+     *   @region?: string,
+     * }|AdminResetUserPasswordRequest $input
+     *
+     * @throws ResourceNotFoundException
+     * @throws InvalidParameterException
+     * @throws UnexpectedLambdaException
+     * @throws UserLambdaValidationException
+     * @throws NotAuthorizedException
+     * @throws InvalidLambdaResponseException
+     * @throws TooManyRequestsException
+     * @throws LimitExceededException
+     * @throws UserNotFoundException
+     * @throws InvalidSmsRoleAccessPolicyException
+     * @throws InvalidEmailRoleAccessPolicyException
+     * @throws InvalidSmsRoleTrustRelationshipException
+     * @throws InternalErrorException
+     */
+    public function adminResetUserPassword($input): AdminResetUserPasswordResponse
```

Comment:
Could you please add an entry in the changelog?

Reply:
Sure, added :)